Source/WebInspectorUI/UserInterface/Views/View.js

```diff
@@ -157,6 +157,7 @@ WI.View = class View extends WI.Object
     {
         this._setLayoutReason(layoutReason);
         this._layoutSubtree();
+        this.parentView?.didLayoutSubtree();
```

Comment:
NIT: Id use `this._parentView` since we're inside the same class and can therefore avoid having to invoke a `get`

Reply:
Fixed